Assets/Scripts/TurnManager.cs

```diff
@@ -57,8 +57,8 @@ void Awake()
 
         foreach (var player in playerList)
         {
-            player.hasMoved = false;
-            player.GetComponentInChildren<MeshRenderer>().material.color = Color.white;
+            players.hasMoved = false;
+            players.gameObject.GetComponentInChildren<MeshRenderer>().material.color = Color.white;
```

Comment:
Wrong variable name here. It is 'player' not 'players'

Reply:
huh.  my pull had that working.  Want me to fix that right now?